pkg/build/controller/policy/serial.go

```diff
@@ -20,7 +20,7 @@ type SerialPolicy struct {
 func (s *SerialPolicy) IsRunnable(build *buildapi.Build) (bool, error) {
 	bcName := buildutil.ConfigNameForBuild(build)
 	if len(bcName) == 0 {
-		return false, NewNoBuildConfigLabelError(build)
+		return true, nil
```

Comment:
hrm, does this mean that creating a build directly will cause the build to schedule immediately?